webcomponents/tool/src/main/frontend/js/sakai-search.js

```diff
@@ -163,119 +169,129 @@ class SakaiSearch extends SakaiElement {
-                  case "ArrowDown":
-                    if (el.nextElementSibling.classList.contains("searchResults")) {
-                      el.nextElementSibling.querySelector("a").focus();
-                      el.nextElementSibling.querySelector("a").classList.add("active");
-                      el.querySelector("a").classList.remove("active");
-                      ke.preventDefault();
-                    }
-                    break;
-                  case "ArrowUp":
-                    if (el.previousElementSibling.classList.contains("searchResults")) {
-                      el.previousElementSibling.querySelector("a").focus();
-                      el.previousElementSibling.querySelector("a").classList.add("active");
-                      el.querySelector("a").classList.remove("active");
-                      ke.preventDefault();
-                    }
-                    break;
-                  default:
-                }
-              });
+          document.querySelectorAll(".searchResults").forEach(el => {
+            el.addEventListener("keydown", ke => {
+              ke.stopPropagation();
+
+              switch (ke.code) {
+                case "ArrowDown":
+                  if (el.nextElementSibling.classList.contains("searchResults")) {
+                    el.nextElementSibling.querySelector("a").focus();
+                    el.nextElementSibling.querySelector("a").classList.add("active", "text-white");
+                    el.querySelector("a").classList.remove("active", "text-white");
+                    ke.preventDefault();
+                  }
+
+                  break;
+
+                case "ArrowUp":
+                  if (el.previousElementSibling.classList.contains("searchResults")) {
+                    el.previousElementSibling.querySelector("a").focus();
+                    el.previousElementSibling.querySelector("a").classList.add("active", "text-white");
+                    el.querySelector("a").classList.remove("active", "text-white");
+                    ke.preventDefault();
+                  }
+
+                  break;
+
+                default:
+              }
             });
           });
-          if (!this.tool) {
-            sessionStorage.setItem("searchresults", JSON.stringify(this.results));
-          }
-          this.requestUpdate();
-        })
-        .catch(error => console.error(error));
-    } else {
-      this.clear();
+        });
+
+        if (!this.tool) {
+          sessionStorage.setItem("searchresults", JSON.stringify(this.results));
+        }
+
+        this.requestUpdate();
+      }).catch(error => console.error(error));
     }
   }
 
   handleButtonClick() {
     const searchButton = this.querySelector("input");
-    searchButton.dispatchEvent(new KeyboardEvent("keydown", {keyCode: 13}));
+    searchButton.dispatchEvent(new KeyboardEvent("keydown", {
+      keyCode: 13
+    }));
   }
 
   initSetsOfResults(results) {
-
```

Comment:
I like to leave a blank line in js functions if there are multiple lines in function body. I think it makes the code clearer.